src/Nethermind/Nethermind.Facade/Eth/RpcTransaction/AuthorizationListForRpc.cs

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using System.Linq;
+using Nethermind.Core;
+using Nethermind.Int256;
+using System.Text.Json.Serialization;
+using System.Text.Json;
+using System;
+using Nethermind.Core.Crypto;
+
+namespace Nethermind.Facade.Eth.RpcTransaction;
+
+[JsonConverter(typeof(JsonConverter))]
+public class AuthorizationListForRpc
+{
+    private readonly IEnumerable<Tuple> _tuples;
+
+    [JsonConstructor]
+    public AuthorizationListForRpc() { }
+
+    private AuthorizationListForRpc(IEnumerable<Tuple> tuples)
+    {
+        _tuples = tuples;
+    }
+
+    private class Tuple
+    {
+        public UInt256 ChainId { get; set; }
+        public ulong Nonce { get; set; }
+        public Address Address { get; set; }
+
+        // TODO: Are these fields actually nullable
+        public UInt256? YParity { get; set; }
+        public UInt256? S { get; set; }
+        public UInt256? R { get; set; }
```

Comment:
@ak88 are these fields actually nullable or can we just default them to `0`?

Reply:
No they cannot be null

Reply:
Removed nullability.